instagram/src/components/Comment/Comment.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import '../Comment/Comment.css';
+
+class Comment extends React.Component {
+  render() {
+    return (
+      <div className={'commentItem'}>
```

Comment:
Your `classNames` don't need to wrapped with curly braces. Anything wrapped in curly braces in JSX will be evaluated as a JS expression. If you were generating classNames dynamically that'd be different, but in that case, you'd be passing a variable without quotes into your curly braces.